shared/schemas/LSP8IdentifiableDigitalAsset.json

```diff
@@ -0,0 +1,51 @@
+[
```

Comment:
it makes me think wheter or not we should also put this schema here:

https://github.com/ERC725Alliance/erc725.js/tree/develop/schemas

I think yes bc why not?

@CallumGrindle what do you think?

Reply:
yeah def! Good idea

Reply:
I mentioned this a while ago and created ticket as well https://app.clickup.com/t/2645698/DEV-5997